util/manual_utils_test.ts

```diff
@@ -41,3 +46,53 @@ Deno.test("get edit link new repo", () => {
     "https://github.com/denoland/manual/blob/v1.12.1/introduction.md",
   );
 });
+
+Deno.test("tocGen", () => {
+  const toc = {
+    "basics": {
+      "name": "Basics",
+      "children": {
+        "import_export": "Importing and Exporting Modules",
+        "node": {
+          "name": "Importing npm modules",
+          "children": {
+            "npm_specifiers": "Using Node.js Packages with npm Specifiers",
+            "cdns": "Using Node.js Packages Via CDNs",
+            "std_node": "The std/node Library",
+            "import_maps": "Using Import Maps",
+            "faqs": "Frequently Asked Questions",
+          },
+        },
+        "standard_library": "Standard Library",
+        "linking_to_external_code": {
+          "name": "Using Third Party Modules",
+          "redirectFrom": ["/linking_to_external_code"],
+        },
+        "permissions": "Permissions",
+        "connecting_to_databases": "Connecting to Databases",
+        "env_variables": "Environment Variables",
+        "testing": {
+          "name": "Testing",
+          "children": {
+            "assertions": "Assertions",
+            "coverage": "Coverage",
+            "documentation": "Documentation",
+            "sanitizers": "Sanitizers",
+            "behavior_driven_development": "Behavior-Driven Development",
+            "mocking": "Mocking",
+            "snapshot_testing": "Snapshots",
+          },
+        },
+        "debugging_your_code": "Debugging Your Code",
+      },
+    },
+    "help": {
+      "name": "Help",
+    },
+  };
+  const { redirectList } = generateToC(toc, "foo");
```

Comment:
Please add some assertions for `pageList` as well

Reply:
done